src/Apps/MyCollection/Routes/EditArtwork/Components/MyCollectionArtworkFormImages.tsx

```diff
@@ -80,6 +114,32 @@ export const MyCollectionArtworkFormImages: React.FC<MyCollectionArtworkFormImag
     )
   }
 
+  const onImgLoad = (
+    image: React.SyntheticEvent<HTMLImageElement, Event>,
+    photoID: string
+  ) => {
+    const {
+      naturalHeight: height,
+      naturalWidth: width,
+      currentSrc,
+    } = image.target as any
```

Comment:
**non-blocking:** Is there any way to avoid `any` here? I think we should try to void the use as much as possible.

Reply:
I don't know how to do that actually - I get Property 'naturalWidth' does not exist on type 'EventTarget'.ts(2339). probably target types are not up to date 🤷‍♂️

Reply:
Have you tried to update it to `image.target as WhatEverTypeIsExpected`? Not perfect but like this we can avoid `any`.

Reply:
I can try it out maybe 👀